app/templates/src/test/javascript/spec/app/account/activate/_activate.controller.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-describe('Controllers Tests ', function() {
+describe('Controller Tests', function() {
```

Comment:
There is another describe in the file called "ActivationController". I can rename that.